src/enclave-agent/Cargo.toml

```diff
@@ -20,7 +20,7 @@ ctrlc = { version = "3.0", features = ["termination"] }
 tokio = { version = "1.14.0", features = ["full"] }
 async-trait = "0.1.42"
 
-image-rs = { git = "https://github.com/confidential-containers/image-rs", tag = "v0.2.0", features = ["occlum_feature"], default-features = false }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", features = ["occlum_feature"], default-features = false, rev = "771e19820ad04946b28b590c3371cf1d765bd14d" }
```

Comment:
For Kata Containers we've bumped the image-rs to cf1f7f96eb60ec4cc4aaa671c04d574a4ea0e441, as that brings in the image layer stream pulling support.

It may be worth doing the same right now.

This will require a bump on the rust toolchain version to the 1.66.0